typedmodels/models.py

```diff
@@ -436,9 +437,16 @@ def save(self, *args, **kwargs):
             raise RuntimeError("Untyped %s cannot be saved." % self.__class__.__name__)
         return super(TypedModel, self).save(*args, **kwargs)
 
-    def _get_unique_checks(self, exclude=None):
+    def _get_unique_checks(self, exclude=None, include_meta_constraints=False):
+        unique_check_kwargs = {"exclude": exclude}
+        parent_unique_checks_args = inspect.getargspec(super(TypedModel, self)._get_unique_checks)
```

Comment:
it's cool that this introspection is possible, but a simple check on `django.VERSION` would be more greppable; makes it clearer when we can remove this code in future...

However, wouldn't this be much simpler if this method just defined `**kwargs` and passed them through to the super call?

Reply:
Indeed, good call